components/post-card/PostCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Avatar,
+  ButtonIcon,
+  EAvatarSizes,
+  EButtonTypes,
+  EIConTypes,
+  EParagraphSizes,
+  Icon,
+  Paragraph,
+} from '@ost-cas-fee-adv-23-24/elbmum-design';
+import { formatDistance } from 'date-fns';
```

Comment:
e.g. formatDistance, decodeTime functions are not used in this file. you can remove their import statements

Reply:
was cleared too.